spec/features/idv/flow_spec.rb

```diff
@@ -347,7 +347,10 @@
     end
 
     scenario 'pick USPS address verification' do
-      sign_in_and_2fa_user
+      issuer = 'http://localhost:3000'
+      user = sign_in_and_2fa_user
+      loa3_sp_session(issuer: issuer)
```

Comment:
This test assumes that the USPS confirmation is performed in a session where SP info is still present, which is not a likely scenario. What do you think about modifying this spec such that the SP session is not present?